packages/api/__tests__/api/private/author.graphql

```diff
@@ -52,5 +52,7 @@ mutation UpdateAuthor($id: ID!, $input: AuthorInput!) {
 }
 
 mutation DeleteAuthor($id: ID!) {
-  deleteAuthor(id: $id)
+  deleteAuthor(id: $id) {
+    ...FullAuthor
```

Comment:
Why did you add this? Consistency?

Reply:
Deletion returns the old object now instead of `id` or `null`. Allows to undo deletions.